spinalcordtoolbox/deepseg/models.py

```diff
@@ -103,6 +103,14 @@
         "description": "SC/GM multiclass segmentation on T2*-w contrast at 7T. The model was created by N.J. Laines Medina, V. Callot and A. Le Troter at CRMBM-CEMEREM Aix-Marseille University",
         "contrasts": ["t2star"],
         "default": False,
+    },
+     "model_seg_epfl_t2w_lumbar_sc": {
```

Comment:
This change will help to fix one of the errors from the [failed linting check](https://github.com/spinalcordtoolbox/spinalcordtoolbox/runs/6011424804?check_suite_focus=true):

```suggestion
    },
    "model_seg_epfl_t2w_lumbar_sc": {
```